### PR TITLE
[filter] Add a time duration log of filter open @open sesame 12/10 12:27

### PR DESCRIPTION
**Pull request by anyj0527**
- Some filter using hardware accelerator consume quite time for loading model file.
- Add log for what time it takes.

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: gst/nnstreamer/tensor_filter/tensor_filter_common.c [hunk 1]
```diff
@@ -2425,6 +2427,11 @@ gst_tensor_filter_common_open_fw (GstTensorFilterPrivate * priv)
     } else {
       priv->prop.fw_opened = TRUE;
     }
+
+    stop_time = g_get_monotonic_time ();
+    ml_logi ("Filter %s with model filed %s is opened. It took %"
```
[the remark this answers is not in the document]
Comment (anyj0527) [reply gst/nnstreamer/tensor_filter/tensor_filter_common.c]: Thanks, it's updated.

**Review by gichan-jang — APPROVED** [review]
LGTM

File: gst/nnstreamer/tensor_filter/tensor_filter_common.c [hunk 2]
```diff
@@ -2425,6 +2427,13 @@ gst_tensor_filter_common_open_fw (GstTensorFilterPrivate * priv)
     } else {
       priv->prop.fw_opened = TRUE;
     }
+
+    stop_time = g_get_monotonic_time ();
+    if (priv->prop.fwname && priv->prop.model_files) {
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_filter/tensor_filter_common.c:2432]: No need to check `priv->prop.fw_opened == TRUE`?

**Review by myungjoo — APPROVED** [review]
LGTM. Please consider @dongju-chae 's suggestion.

File: gst/nnstreamer/tensor_filter/tensor_filter_common.c (hunk 2 above)
Comment (anyj0527) [reply gst/nnstreamer/tensor_filter/tensor_filter_common.c]: Right, No need to log when the opening fails. I added the condition, Thanks!

**Review by dongju-chae — APPROVED** [review]
LGTM

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**